docs/api-docs/partner/subscriptions/foundation-guide.md

```diff
@@ -68,99 +73,111 @@ To develop and test the app, you'll install it on your [sandbox store](#configur
+
+1. On your Stripe Dashboard click **Connect** to enable [Stripe Connect for Platforms](https://dashboard.stripe.com/test/connect/accounts/overview). 
 
 2. Click **Get started**.
 
-3. Select **Platform or Marketplace**, then click the **Continue** button at the lower right.
+3. Select **Platform or Marketplace**, then click **Continue**.
+
+4. To configure the app-specific account's Stripe Connect settings, on your Stripe Dashboard go to [**Settings > Connect settings**](https://dashboard.stripe.com/test/settings/connect). 
 
-4. To configure the app-specific account's Stripe Connect settings, go to [**Settings > Connect settings**](https://dashboard.stripe.com/test/settings/connect). Under **Integration** perform the following steps:
+5. Under **Integration** do the following:
    
-   a. Copy the **Test mode client ID**. In a later step, you will use the client ID to update `NEXT_PUBLIC_STRIPE_CLIENT_ID` in the .env file.
+   a. Copy the **Test mode client ID** and make sure to store it at a safe place. You will use the client ID in a later step to update `NEXT_PUBLIC_STRIPE_CLIENT_ID` in the .env file.
 
    b. Set **OAuth settings** to **OAuth for Standard accounts**. 
 
-   c. Add **Redirects**. Click the **+ Add URI** button and add your app's callback URI.
+   c. Click **Redirects**, and then click **+ Add URI**. Add the callback URI for your app:
    
    ```http title="Your app's Stripe callback URI"
    https://{ngrok_id}.ngrok.io/stripe/callback
    ```
 
 ## Configure the BigCommerce store
 
-To configure the store to make subscription charges, take the following steps:
+After you have successfully created a test configuration, you need to configure your BigCommerce store in the BigCommerce store control panel.
+
+**To configure the store to make subscription charges**
 
-   - In the [Stripe settings section](https://login.bigcommerce.com/deep-links/settings/payment/stripev3) of the BigCommerce store control panel, make sure **Test Mode** is set to **Yes**.
+1. In the BigCommerce store control panel, in the [Stripe settings section](https://login.bigcommerce.com/deep-links/settings/payment/stripev3), make sure that **Test Mode** is set to **Yes**.
    
    ![stripe-settings](https://storage.googleapis.com/bigcommerce-production-dev-center/images/stripe-settings.png)
    
-   - To start a subscription, shoppers must check out using a stored card. To turn on that functionality, go to [Stripe settings](https://login.bigcommerce.com/deep-links/settings/payment/stripev3) in the BigCommerce store control panel, find the **Stored Credit Cards** section, and toggle on **Enable stored credit cards with Stripe**. 
+2. To start a subscription, shoppers must check out using a stored credit card. To enable this functionality, go to [Stripe settings](https://login.bigcommerce.com/deep-links/settings/payment/stripev3), locate the **Stored Credit Cards** section, and toggle on **Enable stored credit cards with Stripe**. 
```

Comment:
`stored card` is a specific term in our literature. Not all stored cards are credit cards; we have stored debit cards and PayPal accounts, for example. If we really want to ax `stored card`, we could say `stored payment instrument`. See the Payment Processing doc.